website/content/docs/concepts/domain-model/index.mdx

```diff
@@ -7,94 +7,129 @@ description: |-
+The diagram below illustrates the relationship between the different levels of scopes.
+
 
 ![](/img/boundary-domain-model-overview.jpg)
 
-## Resources
+The highest-level  scope is the [Global Scope](/docs/concepts/domain-model/scopes#global). Boundary administrators operate on this level to configure and manage Boundary for the entire business. Boundary administrators  can also manage child scopes, auth methods, users, groups, roles, and grants.
+
+The intermediate scope level is  [Organizations](/docs/concepts/domain-model/scopes#organizations), also referred to as orgs, which are a child scope of global.  Though the implementation may be flexible for organization scopes, we recommend scoping by business units in this case. Scoping by business unit provides a clear separation of concerns for each business unit. Creating organizations for the development, operations, and support teams would be an example.
+
+The lowest scope level is the [Project](/docs/concepts/domain-model/scopes#projects). An organization’s scope may have as many projects as required based on its business workflows. To expand on the example used earlier, the development team’s organization can also have project scopes for their different products.
+
+For example, they could have the following project scopes:
+
+- Product 1
+- Product 2
+- Product 3
+
+## Identity and Access Management
+
+As mentioned earlier, IAM  is handled at the organization and global scope levels. Here, operators can create and manage principals. Principals are entities in Boundary that are either [Users](/docs/concepts/domain-model/users) or [Groups](/docs/concepts/domain-model/groups) that may be assigned capabilities.
+
+Users are associated with either the global or an organization’s scope. For better access control management, users may be positioned into groups to ease access  control demands. Groups can also contain users from different organizations or global scopes without creating multiple users for a single entity, making permissioning less complicated to implement.
+
+An [Auth Method](/docs/concepts/domain-model/auth-methods) is the mechanism in Boundary used for authentication. An organization may use numerous authentication methods. Boundary supports Open ID Connect (OIDC) auth methods to delegate authentication to an external identity provider (IDP) - such as Azure Active Directory, Okta, and other OIDC providers - as well as a Boundary-native password auth method. Within an auth method, [Accounts](/docs/concepts/domain-model/accounts) may be created and associated with users.  Once associated  with a user, the principal may authenticate in Boundary using the method specified by their auth method. An account for the password auth method consists of a login name and password.
```

Comment:
Could be useful to provide an example of a user having multiple accounts. For example, being connected with OIDC as one but also having a username-password in break-glass scenarios.

Reply:
@PPacent , can you help fill in the content gaps here per xingluw's suggestions? Thanks.

Reply:
@taoism4504 you can use the following:

"A user may be associated with accounts from different auth methods, such as a user having both an Azure Active Directory account and an Okta account. In this case, the user could authenticate with either provider."